README.md

```diff
@@ -37,8 +37,7 @@ server:
   compress: false
   header: true
   serveStatic:
-    extensions:
-    - html
```

Comment:
Removed as they are not the default. User can always refer to https://github.com/expressjs/serve-static#options.